### DISCUSSION TITLE
Using multiple samplers

### DISCUSSION THREAD

**Discussion by mikhael-cutajar** [TorchIO-project/torchio#951] [Q&A]
Hi, I am an unsure if it is possible to use 2 different samplers in a patch based pipeline. I would like to create a queue which includes patches extracted using the label sampler and the uniform sampler. Is this possible? Thanks

**Comment by fepegar** [comment]
Hi, @mikhael-cutajar. Apologies for the delayed response. I'm not sure this is currently possible. I guess you could subclass the `Queue`, modifying it to take multiple samplers.

Closing this discussion as stale, feel free to reopen if needed.
